ws-nextjs-app/pages/[service]/live/[id]/Post/index.tsx

```diff
@@ -150,14 +150,14 @@ const PostContent = ({ contentBlocks }: { contentBlocks: OptimoBlock[] }) => {
       />
     ),
     video: (props: ComponentToRenderProps) =>
-      isLive() ? (
-        <LegacyMediaPlayer
+      isTestRequested() ? (
```

Comment:
Should we maybe add a couple of test scenarios for this, just to make sure that we have the right combination of environment / renderer_env? (Unless we need to move quickly to get this merged - if so, could we do a follow up PR to add the tests?)